src/GenreBar.js

```diff
@@ -10,21 +10,31 @@ export default class GenreBar extends Component {
     }
     updateGenres = () => {
         let checked = [];
-        const checkboxes = document.querySelectorAll('.genreCheckBox:checked')
-            checkboxes.forEach((elem) => {
-            checked.push(elem.value)
-    })
-    this.setState({genresToFilter: checked});
-    this.props.updateState({genresToFilter: this.state.genresToFilter})
+        const checkboxes = document.querySelectorAll('.genre-check-box')
```

Comment:
We want to avoid doing our own DOM manipulation/traversal with vanilla JavaScript. It looks like you have an `onChange` event on your `form` element down below --- it would more closely align with React patterns to have an `onChange` for each CheckBox component that pushed/popped the genre/platform into your `genresToFilter` and `platformsToFilter` arrays.